src/NexusMods.App/Program.cs

```diff
@@ -148,6 +152,48 @@ public static int Main(string[] args)
         }
     }
 
+    private static async Task CleanupOldProcesses(IServiceProvider serviceProvider)
```

Comment:
Function should be renamed to something more accurate like `CheckForExistingProcess`, since if another process is alive and responding it wouldn't get cleaned up.

```suggestion
    private static async Task CheckForExistingProcess(IServiceProvider serviceProvider)
```

Reply:
`CheckForExistingProcess` is even less accurate since it doesn't indicate that the existing process will be killed, which is pretty important. I can rename to `CleanupUnresponsiveProcesses`.